packages/@aws-cdk/core/lib/stage.ts

```diff
@@ -137,6 +146,15 @@ export class Stage extends Construct {
    */
   private assembly?: cxapi.CloudAssembly;
 
+  /**
+   * Validation plugins to run during synthesis. If any plugin reports any violation,
+   * synthesis will be interrupted and the report displayed to the user.
+   *
+   * @default - no validation plugins are used
+   */
+  public readonly policyValidation: IPolicyValidationPluginBeta1[] = [];
```

Comment:
```suggestion
  public readonly policyValidationBeta1: IPolicyValidationPluginBeta1[] = [];
```

Otherwise changing the type to `IPolicyValidationPluginBeta2` would be a breaking change.